otelcol.exporter.prometheus add_metrics_suffixes option (#5568)
grafana/agent: CHANGELOG.md

- Improved performance of `pyroscope.scrape` component when working with a large number of targets. (@cyriltovena)

> - Added new config options to `otelcol.exporter.prometheus` in flow mode (@mar4uk):
>   - `add_metric_suffixes`: configures whether to add type and unit suffixes to metrics names.

Reviewer: ```suggestion
- Added an `add_metric_suffixes` option to `otelcol.exporter.prometheus` in flow mode, which configures whether to add type and unit suffixes to metrics names. (@mar4uk)
```
